Wireframe/index.html

```diff
@@ -4,10 +4,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
```

Comment:
It would be better to use a better more descriptive title.

Reply:
I change Wireframe to GitHub Tutorial